app/src/components/Card.jsx

```diff
@@ -0,0 +1,12 @@
+import "./Card.css";
+import SocialMediaButtons from "./SocialMediaButtons.jsx";
+import InformationAboutMe from "./InformationAboutMe.jsx";
+const Card = () => {
+  return (
+    <div class="card">
+      <InformationAboutMe />
+      <SocialMediaButtons />
```

Comment:
This structure is not really reusable for this app. We should define some bricks from which we can build our house. Instead you started to build this house from everything what was near the hands. Its okay by the way, looks nice. Just lest make it even more nicer

The plan.
Seems like we have a few entities here:
1. Avatar
2. Button
3. Link
4. Container
5. Text
6. Card (which is constructed from everything above)


I believe it is not a bad idea to create this 6 components and incapsulate our app a little bit. Which is gonna give you more flexibility in case you want to have some changes in design and the logic

Here is some basic requirements for those components you are about to create:
1. `Avatar.jsx` - gets a src of the image and any of 3 sizes (s,m,l) - on exit it gives you avatar with its declared size. if size is not set, lets make it `m` by default
2. `Button.jsx` - optional parameter `onClick` + text as component `children` prop
3. `Link.jsx` should get a `href` parameter and `children` parameter which can be anything - text, number, component. Also check out how `target="_blank"` works for links. And add this to the params of this component
4. `Container.jsx` - lets make it just a flex-box container for the beginning. It should have all basic flex-box properites: `flexStart, flexEnd, justifyContent, alignItems, flexGrow, flexShrink`
5. `Text.jsx` - `size`, `color`, `children`
6. `Card.jsx` - once you have done with everything above you can easy peasy create this component like lego contsructor

Reply:
Nice! Please also add `Container.jsx` and apply it in `Card.jsx`